packages/components/src/Drawer/Drawer.component.js

```diff
@@ -23,12 +24,16 @@ DrawerAnimation.propTypes = {
 	children: PropTypes.node,
 };
 
-function DrawerContainer({ stacked, className, children, ...rest }) {
+function DrawerContainer({ stacked, className, children, withoutTransition = false, ...rest }) {
 	const drawerContainerClasses = classnames(
 		theme['tc-drawer'],
 		className,
 		'tc-drawer',
 		{
+			[theme['tc-drawer-transition']]: !withoutTransition,
```

Comment:
It's usually better to try and avoid double negatives. They create confusions on the conditional flow.
Would be better to rename `withoutTransition` to `withTransition` defaulting to `true`.